src/main/java/org/logstash/netty/SslContextBuilder.java

```diff
@@ -52,7 +52,9 @@
             "TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384",
             "TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384",
             "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256",
-            "TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256"
+            "TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256",
+            "TLS_AES_128_GCM_SHA256",
+            "TLS_AES_256_GCM_SHA384",
```

Comment:
believe these should be at the top (still unsure about trying to detect the cha-cha ciphers)

Reply:
Fixed.  

I also added proposal for auto-detecting cha-cha: All the ciphers are now in `DEFAULT_CIPHERS` but when `getDefaultCiphers()` is called it returns union of default and what is supported by current Java version.  I guess documentation should then have two versions of defaults (not updated yet)?

I added some kind of minimal test case for that too and noticed that the existing TLS Java tests assumed certificate in `/tmp`, so I added one test cert in test resources from [TCP input fixtures](https://github.com/logstash-plugins/logstash-input-tcp/tree/main/spec/fixtures).

I can remove this auto-detect stuff if it does not sound like a good idea after all, I don't mind!

Reply:
>  also added proposal for auto-detecting cha-cha: All the ciphers are now in `DEFAULT_CIPHERS` but when `getDefaultCiphers()` is called it returns union of default and what is supported by current Java version.

believe, this is fine for now - some kind of filtering as the best we could do anyways.

> I guess documentation should then have two versions of defaults (not updated yet)?

I think we can have just one set not to confuse users - let's include the CHA-CHA ciphers and have a note in the documentation that mentions that the list is accurate for Java 11+